### ISSUE TITLE
List of current symbols and codepoints

### ISSUE THREAD

**Issue by powerman** [ryanoasis/nerd-fonts#144]
This list is useful for testing and choosing interesting symbols.
Also it's more consistent with current fonts than codepoints shown in README.
So, it may worth to add this to the wiki - I've opened issue just to ask is this appropriate. Probably it's better to put it on page with Nerd font loaded.

## IEC Power Symbols (23FB-23FE,2B58)
```
⏻⏼⏽⏾⭘
```

## Font Awesome Extension (moved E000-E0A9 → E200-E2A9)
```





```

## Powerline Extra Symbols (E0A0-E0A3 E0B0-E0BF E0C0-E0C8 E0CC-E0CF E0D0-E0D2 E0D4)
```

```

## Devicons (moved E600-E6C5 → E700-E7C5)
```





```

## Font Awesome (F000-F2E0 with holes)
```

















```

## Octicons (2665,26A1, moved F000-F105 with holes → F400-F4A8, moved F27C → F67C)
```
♥⚡





```

## Pomicons (E000-E00A)
```

```

## Font Linux (moved F100-F115 with holes → F300-F313)
```

```

## Seti-UI + Custom (E5FA-E62B)
```


```

**Comment by ryanoasis** [comment]
@powerman Hey thanks for this! Really great :smile: 

Just FYI trying to build a script for output/testing, you can see some progress here: https://github.com/ryanoasis/nerd-fonts/issues/84#issuecomment-288576052

Yeah if this was on the wiki I think it would have to be as images but you are right using it on a page with the font actually loaded would be very helpful!

Perhaps on the website (http://nerdfonts.com/) It's pretty bare-bones right now but I do envision it having the glyphs there and even searchable. The site is currently living in the [gh-pages branch](https://github.com/ryanoasis/nerd-fonts/tree/gh-pages)

**Comment by powerman** [comment]
I've updated list - some Awesome icons was missing, not sure how that happens. :(

**Comment by msva** [comment]
Uhm... I did a simple script on Lua to test fonts... And I have few questions:
![screenshot_20170331_105543](https://cloud.githubusercontent.com/assets/268102/24535861/fb407874-1600-11e7-8e7f-8cbb2690212c.png)

1) Does that glyphs rendered under "Seti UI" font really what is intnder to be rendered there?
2) Take a look on glyphs between red `[` and `]`. Also take a look at the underlined glyph. As far as I see, first ones is rendered "too low". Is it any way to fix that? Or was that intended?
3) Take a look on Powerline Extra Symbols. There is "flame" glyphs rendered pretty fine, but "lego" glyphs looks like cut at bottom. Aren't flame glyphs drawn a bit higher in source glyphs?

**Comment by ryanoasis** [comment]
@msva Great nice script. Feel free to submit a Pull Request to add it to Nerd Fonts :smile: 

1. Are you referring to the glyph at `e613` yes it was the original folder from Seti UI. The other folders I added and customized from other sources (in particular to have an open folder as well).
2. Yes those look to be too low as you mention. It was not intended no. It should hopefully be fixed in [v1.1.0 Release](https://github.com/ryanoasis/nerd-fonts/milestone/16) (see below for a preview). You can also look into: #142
3. Yeah I see that problem, again hopefully we can fix in [v1.1.0](https://github.com/ryanoasis/nerd-fonts/milestone/16)

### By the way which font were you using in your screenshot?

## Preview of fixes coming in [v1.1.0](https://github.com/ryanoasis/nerd-fonts/milestone/16)*

| **Before fix (left side)** | **After fix (right side)** |
|--------------------------------------|--------------------------------------|
| ![selection_022-prefix](https://cloud.githubusercontent.com/assets/8083459/24581660/9e365e94-16ed-11e7-81ab-2341b3829c5f.png) | ![selection_021-postfix](https://cloud.githubusercontent.com/assets/8083459/24581663/a96e9d26-16ed-11e7-8a8b-8194963e432c.png) |

* Still a Work-In-Progress

**Comment by msva** [comment]
0)
> Great nice script <...> submit

As I said, it's in Lua. Not sure, people will want to install lua (even it is very small and has very small runtime memory footprint) to just test fonts :-D
And also, I want to try to rewrite it to render a table, like you showed in another issue :)

But... If you're really sure Lua script will be acceptable, I'll definitely PR it when it'll be done.

// I'd maybe try to rewrite it on bash, but there is not possible to use arrays of arrays, and doing that magic in other way is a few times harder :)

1)
> Seti-UI

Actually, I talking mostly about that greek letters, but it looks like that thing is related to the font (they do exist in original font).
And I'd very like to somehow force Nerd Symbols for some glyph ranges, but I can't find proper way for fontconfig to do that (I did very black magic there even without that condition).

So, if anybody knows how to force some font for some glyph ranges in fontconfig's xml, I'd be very happy if you'll share that knowledge.

4)
> By the way which font were you using in your screenshot?

Actually, it is `PragmataPro` (not even `... Mono` one). Since it is monospaced anyway, I forced it (with fontconfig) to be used in terminal instead of it's Mono version (which has ligatures removed).

**Comment by msva** [comment]
By the way, @ryanoasis, maybe you know how to remove glyphs by their codes from the multiple the fonts in a row (i.e. not opening FontForge gui, but script it in some way)?

// although, it throws some warnings and errors on some fonts, and I afraid it will brake something after such changes :-/

**Comment by ryanoasis** [comment]
@msva not sure exactly but you can probably find a decent fontforge python script on github

**Comment by mr-ubik** [comment]
Is there any plan on updating the README pics to reflect the codepoints change?

**Comment by ryanoasis** [comment]
@mr-ubik Sorry for any current confusions. To answer your question yes, see: #149

**Comment by mr-ubik** [comment]
@ryanoasis thanks for the reply. Is there a list of things that are still missing / you need help with?

**Comment by ryanoasis** [comment]
@mr-ubik Sure no problem :smile: I am going to assume you mean help in general (not with the codepoints specifically):

Things that need help:

- In general any ideas, PRs, bug reports, testing is appreciated :smile: 
- [the wiki](https://github.com/ryanoasis/nerd-fonts/wiki), in particular:
  - [Terminal Emulators](https://github.com/ryanoasis/nerd-fonts/wiki/Terminal-Emulators)
  - [Developer Setup macOS/osx section](https://github.com/ryanoasis/nerd-fonts/wiki/Contributor-Developer-Setup#osx)
  - [Screenshots](https://raw.githubusercontent.com/wiki/ryanoasis/nerd-fonts/screenshots) (not sure if this is worth creating, maybe?)

---

Let me know if I misunderstood your question

**Comment by ryanoasis** [comment]
@powerman I think we can now consider this closed because of combination of merged PR #147 and [Cheet Sheet](http://nerdfonts.com/#cheat-sheet)

Please comment or re-open if you disagree. Thanks!
